desktop/hardened-desktop/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Hardened Desktop
+description: Overview of what Hardened Desktop is
+keywords: security, hardened desktop, enhanced container isolation, registry access management, admin controls, root access, administrators, docker desktop
+---
+>Note
+>
+>Hardened Desktop is available to Docker Business customers only.
+
+Hardened Desktop is a new security model for Docker Desktop. It's designed to provide administrators with a simple and powerful way to improve their organizations security posture for containerised development, without impacting the developer experience that Docker Desktop offers.
+
+This configuration is designed for security conscious organizations who don’t give their users root or admin access on their machines, and who would like Docker Desktop to be within the organization’s centralized control.
+
+The Hardened Desktop security model moves the ownership boundary for containers to the organization, meaning that any security controls administrators set cannot be altered by the user of Docker Desktop.
+
+Hardened Desktop includes:
+- Admin Controls, which helps administrators to confidently manage and control the usage of Docker Desktop within their organization.
+- Enhanced Container Isolation, a setting that instantly enhances security by preventing containers from running as root in Docker Desktop’s Linux VM and ensures that configurations set using Admin Controls cannot be modified by containers.
+- Registry Access Management, which allows administrators to control the registries developers can access.
+
+Docker plans to continue adding more security enhancements to the Hardened Desktop security model.
```

Comment:
@ctalledo @aevesdocker - i wonder if we can add a little more detail here. e.g. `Security is a moving target and as such, Docker will be continuously adding enhancements to Hardened Desktop and it's underlying features'

Reply:
@christophermclellan I wonder if this is actually the place for the sentence at all. This seems more like it belongs in marketing/blogs rather than docs which is meant to represent the 'here and now', rather than the 'what's coming soon'

Reply:
yeh, its a good point. ok let's leave it as is. i think having the original 'docker plans to continue adding...' is still useful as we want to convey that 'this is not completely, 100% hardened, and that we still have some things we want to work on'.